antarest/core/exceptions.py

```diff
@@ -8,6 +8,43 @@ class ShouldNotHappenException(Exception):
     pass
 
 
+class STStorageFieldsNotFoundError(HTTPException):
+    """Fields of the short-term storage are not found"""
+
+    def __init__(self, storage_id: str) -> None:
+        detail = f" fields of storage '{storage_id}' not found"
+        super().__init__(HTTPStatus.NOT_FOUND, detail)
+
+    def __str__(self) -> str:
+        return self.detail
+
+
+class STStorageMatrixNotFoundError(HTTPException):
+    """Matrix of the short-term storage is not found"""
+
+    def __init__(
+        self, study_id: str, area_id: str, storage_id: str, ts_name: str
+    ) -> None:
+        detail = (
+            f" time series '{ts_name}' of storage '{storage_id}' not found"
```

Comment:
you can remove the spacing at start and capitalize the first letter to be consistent with other exceptions 

> Note: same for `STStorageConfigNotFoundError`